docs/Materials.md.html

```diff
@@ -1119,6 +1119,27 @@
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+### General: instanced
+
+Type
+:    `boolean`
+
+Value
+:     `true` or `false`. Defaults to `false`.
+
+Description
+:    Allows a material to access the instance index (i.e.: **`gl_InstanceIndex`**) of instanced
+     primitives using `getInstanceIndex()` in the material's shader code. Never use
+     **`gl_InstanceIndex`** directly. This is typically used with
```

Comment:
Maybe we could add a warning to `matc` if we detect someone using `gl_InstanceIndex` / `gl_InstanceID`

Reply:
that's a good idea. probably same with gl_FragCoord.